time time
int32 num
string status
string satellite
float32 rssi
float32 noise_level